File: Código/FrontEnd/SPODS/src/app/shared/contacto.ts
import { Cliente } from "./cliente";
import { Publicacion } from "./publicacion";
import { ComentarioPuntuacion } from "./comentarioPuntuacion";

export class Contacto {
    Id: number;
    Publicacion: Publicacion = new Publicacion();
    Cliente:Cliente = new Cliente();
    ComentarioPuntuacion: ComentarioPuntuacion = new ComentarioPuntuacion();
    Fecha: Date;
}

File: CD Entrega Vieja/Fuentes/Backend y API Rest/ET/Contacto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ET
{
    public class Contacto
    {
        public int Id { get; set; }
        public Cliente Cliente { get; set; }
        public Publicacion Publicacion { get; set; }
        public ComentarioPuntuacion ComentarioPublicacion { get; set; }
        public DateTime Fecha { get; set; }
    }
}
